package.json

```diff
@@ -8,7 +8,6 @@
     "url": "git+https://github.com/jkuszmaul/flatbuffers_reflection.git"
   },
   "author": "James Kuszmaul",
-  "license": "Apache-2.0",
```

Comment:
Doesn't seem intentional? Unless `license` is not a supported tag in newer npm versions or something?

Reply:
there were two of these :)

Reply:
Well it is Apache *2*.